pkg/serviceactivities/shutdown/shutdown.go

```diff
@@ -38,18 +40,24 @@ func New() *ServiceActivity {
 
 // Run runs the shutdown service activity
 func (s *ServiceActivity) Run(ctx context.Context) error {
-	// listen for interrupts and gracefully shutdown server
-	c := make(chan os.Signal, 1)
-	signal.Notify(c, os.Interrupt, syscall.SIGTERM, syscall.SIGHUP)
+	// listen for interrupt, terminated, and hangup signals and gracefully shutdown server
+	cInt := make(chan os.Signal, 1)
+	signal.Notify(cInt, os.Interrupt)
+	cTerm := make(chan os.Signal, 1)
+	signal.Notify(cTerm, syscall.SIGTERM)
```

Comment:
Use `os.Kill` instead? More portable.

Reply:
Not that the program is going to really be able to handle this anyways...

Reply:
we currently don't trap kill, and i think we can't, because, like you say, that's the end of everything immediately, right?

Reply:
yeah, gdi, i conflated them both again. Ignore me. 😢